ding_dibs.module

```diff
@@ -386,6 +386,28 @@ function ding_dibs_menu() {
     'file' => 'ding_dibs.reports.inc',
   );
 
+  $items['admin/reports/ding_dibs_emails'] = array(
```

Comment:
Thanks for moving the reporting. The configuration still belongs under `admin/config`. The previous path for the settings page `admin/config/payment/ding_dibs/emails` seemed fine to me.